applications/StructuralMechanicsApplication/custom_elements/prestress_membrane_element.cpp

```diff
@@ -134,6 +134,8 @@ void PrestressMembraneElement::Initialize()
     mThickness0 = GetProperties()[THICKNESS];
```

Comment:
in the function calculateAll() I need to integrate on the reference configuration, so I stored the initial thickness when I initialize the element.